travis-ci/managers/debian.sh

```diff
@@ -30,55 +30,41 @@ for phase in "${PHASES[@]}"; do
         SETUP)
             info "Setup phase"
             info "Using Debian $DEBIAN_RELEASE"
-	    docker pull debian:$DEBIAN_RELEASE
+            docker pull debian:$DEBIAN_RELEASE
             info "Starting container $CONT_NAME"
             $DOCKER_RUN -v $REPO_ROOT:/build:rw \
                         -w /build --privileged=true --name $CONT_NAME \
-			-dit --net=host debian:$DEBIAN_RELEASE /bin/bash
+                        -dit --net=host debian:$DEBIAN_RELEASE /bin/bash
             docker_exec bash -c "echo deb-src http://deb.debian.org/debian $DEBIAN_RELEASE main >>/etc/apt/sources.list"
             docker_exec apt-get -y update
             docker_exec apt-get -y build-dep libelf-dev
             docker_exec apt-get -y install libelf-dev
             docker_exec apt-get -y install "${ADDITIONAL_DEPS[@]}"
             ;;
-        RUN|RUN_CLANG|RUN_GCC8)
-            if [[ "$phase" = "RUN_CLANG" ]]; then
+        RUN|RUN_CLANG|RUN_GCC8|RUN_ASAN|RUN_CLANG_ASAN|RUN_GCC8_ASAN)
+            if [[ "$phase" == *"CLANG"* ]]; then
```

Comment:
nit: let's use either `==` or `=` in `if` statement. Imo `=` is better since `==` is a bash-ism.

Reply:
if there is no difference in behavior and it still works (I'll check) then yeah, I'll update before pushing.

Thanks for review!

Reply:
well, it obvious does work because I accidentally used '=' for GCC8 case :)